tests/cache_control.rs

```diff
@@ -32,3 +39,56 @@ async fn lists_existing_test_cache() {
         .collect();
     assert!(cache_names.contains(&cache_name));
 }
+
+#[tokio::test]
+async fn flush_nonexistent_cache_returns_not_found() {
+    let client = CACHE_TEST_STATE.client.clone();
+    let cache_name = "fake-cache-".to_string() + &Uuid::new_v4().to_string();
+    let result = client.flush_cache(cache_name).await.unwrap_err();
+    let _err_msg = "not found".to_string();
+    assert!(matches!(result.to_string(), _err_message))
+}
+
+#[tokio::test]
+async fn flush_existing_cache_returns_success() {
+    let client = CACHE_TEST_STATE.client.clone();
+    let cache_name = CACHE_TEST_STATE.cache_name.clone();
+
+    // Insert some elements
+    let set_result1 = client
+        .set(cache_name.to_string(), "key1", "value1")
+        .await
+        .unwrap();
```

Comment:
we generally want to try to avoid calling `unwrap`. It's better to give the test fns a return type of `MomentoResult<()>`, and then you can use `await?`. The test will need to return `Ok(())` at the end.

Our linter usually fails builds if we use `unwrap` but I guess it doesn't apply that rule to tests. It's not the end of the world to use them here but better to get out of the habit of using them I think.